src/data/frontend-user-data.ts

```diff
@@ -0,0 +1,23 @@
+import { HomeAssistant } from "../types";
+
+export const fetchFrontendUserData = async (
```

Comment:
We have been naming most files to their component name, so this should be `frontend.ts`